uninstall.php

```diff
@@ -11,4 +11,22 @@
 	exit;
 }
 
-delete_option( 'perflab_modules_settings' );
+if ( is_multisite() && 100 >= get_sites( array( 'count' => true ) ) ) {
```

Comment:
I feel like we should check the count condition inside the loop. Otherwise, it will run else part for multisite with more than 100 sites.

Reply:
Also, let's add a comment here explaining this block.
```suggestion
// For a multisite, delete the option for all sites (however limited to 100 sites to avoid memory limit or timeout problems in large scale networks).
if ( is_multisite() ) {
```